python/tests/test_appkit_examples.py

```
@@ -0,0 +1,334 @@
+        "DATABRICKS_TOKEN": "local-test-token",
+        "DATABRICKS_WORKSPACE_ID": "local-test-workspace",
+        "MLFLOW_TRACKING_URI": f"file:{tmp_path / 'mlruns'}",
+        "PYTHONDONTWRITEBYTECODE": "1",
+        "PYTHONPATH": os.pathsep.join(pythonpath_parts),
+    }
+    tool_case = BRIDGE_TOOL_CASES.get(rel)
+    if tool_case is not None:
+        env["APX_APPKIT_TOOL_CASE"] = json.dumps(tool_case)
+    env.pop("DATABRICKS_CONFIG_PROFILE", None)
+
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)